src/assets/constants/constants.ts

```diff
@@ -5,9 +5,9 @@ import { Util } from '../../JSCommon/util';
 export class Constants {
   public static readonly HACKATHON_NAME = 'hackpsuS2019';
   public static readonly sqlConnection = {
-    acquireTimeout: 60 * 60 * 1000,
-    connectTimeout: 60 * 60 * 1000,
-    connectionLimit: 1000,
+    acquireTimeout: 10 * 1000,
+    connectTimeout: 10 * 1000,
+    connectionLimit: 56,
```

Comment:
What is the purpose of decreasing the timeout? Follow up why specifically reduce the connection limit to 56 compared to 50 or 100.

Reply:
on average we run with about 5 instances during the event. the connection limit is 280, so 280/5. The timeout causes less requests to pile up in case of heavy traffic and return 500. This triggers a scaling up instead of some of the bad errors we saw last night.